agent/rpm/pbench-agent.spec.j2

```diff
@@ -89,23 +89,10 @@ cp -a agent/* %{?buildroot}/%{installdir}/
 if pip3 show configtools > /dev/null 2>&1 ;then pip3 uninstall -y configtools ;fi
 
 %post
-%if 0%{?rhel} == 7
-# This is a very ugly work-around for RHEL 7 / CentOS 7 environments. The
-# version of `pip3` that is available in those environments no longer works
-# to install the latest Python3 modules, which we need. So we have to fetch
-# our own copy of `pip3`, as described at:
-#   https://pip.pypa.io/en/latest/installing/#installing-with-get-pip-py 
-# We then install it separately from the existing version and set `PATH` and
-# `PYTHONPATH` so that we use this updated version when we install our module
-# dependencies below.
-curl -X GET -o /%{installdir}/get-pip.py https://bootstrap.pypa.io/get-pip.py > /%{installdir}/pip3-update.log 2>&1
-python3 /%{installdir}/get-pip.py --prefix=/%{installdir} >> /%{installdir}/pip3-update.log 2>&1
-export PYTHONPATH=/%{installdir}/lib/python3.6/site-packages:${PYTHONPATH}
-export PATH=/%{installdir}/bin:${PATH}
-%endif
```

Comment:
I've encountered situations (of unknown cause) on both server and agent installs where this doesn't work; we still get a "no pip command" error and nothing gets installed.

When we butted our heads against this during server jam sessions, we ended up using the new code I've pasted here, which just uses `python3 -m pip` instead of expecting a `pip` command. I really need to experiment with this on some bare untouched systems to be sure it works on them all; which is only one of the reasons this is still a draft. But if it *doesn't* work, then we've got some holes that may be hard to fix.

Reply:
I decided to do what I resisted in the first place, pushing this out of the PR to test separately with the new VM pool. (Once I get through all the `subscription-manager` setup muddle across the releases.) So no longer relevant here...